korgan/payment_runtime.py

```
@@ -131,223 +119,225 @@ async def payment_proof_requested(callback: CallbackQuery, state: FSMContext) ->
-    await message.answer(
-        "📎 Чектің өзін фото, JPG/PNG/WEBP немесе PDF түрінде жіберіңіз."
-        if language == "kk"
-        else
-        "📎 Пришлите сам чек как фото, JPG/PNG/WEBP или PDF."
-    )
-
-
-@router.message(PaymentReceiptFilter())
-async def payment_receipt_received(message: Message, state: FSMContext) -> None:
-    """AI-verifies a receipt and immediately releases/starts the paid document."""
+async def _verify_and_release_fiscal_url(message: Message, state: FSMContext, receipt_url: str) -> None:
```

Ruff (0.16.2)

[warning] Too many return statements (13 > 6)

(PLR0911)